src/data/checkfiles/default_checkfiles/validate-resc-chars.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { Link } from '@reach/router'
+
+export default {
+    name: "Each resource name cannot have invalid characters",
+    description: `Checks if the length of each resource is between 1 and 63 characters, starts & ends with a word character, and does not contain consecutive dashes.`,
+    minimum_server_version: "4.2.0",
+    maximum_server_version: "",
+    interval_in_seconds: 3600,
+    active: true,
+    checker: function () {
+        let result = {
+            status: '',
+            message: '',
+            success: 0,
+            failed: []
+        }
+        const regex = /^(?=.{1,63}$)\w+(-\w+)*$/; // taken from `irods/plugins/database/src/db_plugin.cpp`: https://github.com/irods/irods/blob/1c360003aea23c9bb80025c3c62dc7a0cef29524/plugins/database/src/db_plugin.cpp#LL684C23-L684C49
```

Comment:
style only - the line feels too long
let's move this comment to two lines prior to the const regex.